AudioSystem/Editor/AudioClipContextMenuExtensions.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace DUCK.AudioSystem.Editor
+{
+	public class AudioClipContextMenuExtensions : UnityEditor.Editor
+	{
+		private const string GENERATE_AUDIO_CONFIGS = "Assets/Generate AudioConfig";
+
+		[MenuItem(GENERATE_AUDIO_CONFIGS, true)]
+		private static bool CanGenerateAudioConfig()
+		{
+			return Selection.objects.All(obj => obj is AudioClip);
+		}
+
+		[MenuItem(GENERATE_AUDIO_CONFIGS)]
+		private static void GenerateAudioConfig()
+		{
+			if (!CanGenerateAudioConfig())
+			{
+				Debug.LogError("Unable to generate an AudioConfig from the selected assets.");
+				return;
+			}
+
+			GenerateAudioConfig(Selection.objects);
+		}
+
+		private static void GenerateAudioConfig(UnityEngine.Object[] selectedObjects, bool groupIntoOneConfig = true)
```

Comment:
Does the user ever have the chance to call this function with `groupIntoOneConfig == false`?

You might need another menu item like "GenerateAudioConfigSeparately".

Reply:
It's not possible at the moment, and I didn't end up using it. I thought it wasn't worth adding a new menu item but didn't want to throw away the functionality.